axelor-sale/src/main/java/com/axelor/apps/sale/service/saleorder/SaleOrderLineGeneratorServiceImpl.java

```diff
@@ -124,4 +135,27 @@ protected void checkProduct(SaleOrder saleOrder, SaleOrderLine saleOrderLine, Pr
           product.getName());
     }
   }
+
+  protected void checkMultipleQty(Product product, BigDecimal qty, SaleOrderLine saleOrderLine)
```

Comment:
I think you can remove the SaleOrderLine from the parameters and move this method in ProductMultipleQtyService, it could be re-used somewhere else.